pandas/tests/frame/test_arithmetic.py

```diff
@@ -27,6 +28,16 @@
 )
 
 
+@pytest.fixture(
+    autouse=True, scope="module", params=[0, 1000000], ids=["numexpr", "python"]
+)
+def switch_numexpr_min_elements(request):
+    _MIN_ELEMENTS = expr._MIN_ELEMENTS
+    expr._MIN_ELEMENTS = request.param
+    yield request.param
```

Comment:
I copy/pasted this fixture from the `tests/arithmetic/`. Is there a way to share to such an autouse fixture?

A ``from pandas.tests.arithmetic.conftest import switch_numexpr_min_elements`` would do the trick, but for some reason we disallow explicit imports from `conftest.py` files.

Reply:
i don't think this needs to be autouse if you are using this explicity (which you are), so just put it in the conftest in tests/frame

Reply:
It is autoused by all tests in this file, I didn't add it to each test function signature.

Reply:
> Is there a way to share such an autouse fixture?

only thing that comes to mind is defining it as non-autouse in a high-level conftest and then

```
@pytest.fixture(autouse=True)
def switch_numexpr_min_elements(switch_numexpr_min_elements_non_auotuse):
    yield from switch_numexpr_min_elements_non_autouse()
```

Reply:
I am not sure that can work with a fixture as the base function. Because if a fixture is passed as an argument like this, I think it is already what is yielded from that function that is being passed to the new fixture? (it might be possible, but in any case I don't get it to work)

Reply:
fair enough; its pretty magical.  maybe with `yield switch_numexpr_min_elements_non_autouse` instead of `yield switch_numexpr_min_elements_non_autouse()`?